lib/parsers/index.ts

```diff
@@ -35,7 +35,8 @@ export interface PkgTree {
     nodeVersion: string;
   };
   labels?: {
-     scope: Scope;
+     scope?: Scope;
+     [key: string]: string | undefined;
```

Comment:
We don't want this to be more strict typed? Or we want to allow custom labels ?

Reply:
we do, changing